oak_restricted_kernel_interface/src/syscalls.rs

```diff
@@ -52,7 +52,8 @@ pub enum Syscall {
     ///   - arg0 (*const c_void): hint for start address for the new mapping, may be nullptr
     ///   - arg1 (c_size_t): size of the new mapping
     ///   - arg2 (c_int): protection on mapping (PROT_EXEC, PROT_READ, PROT_WRITE, PROT_NONE)
-    ///   - arg3 (c_int): flags. The only combination we support is MAP_PRIVATE | MAP_ANONYMOUS.
+    ///   - arg3 (c_int): flags. The only combination we support is MAP_PRIVATE | MAP_ANONYMOUS +
+    ///     MAP_FIXED.
```

Comment:
I am not 100% sure what this notation implies. Does it mean that both the MAP_PRIVATE and MAP_ANONYMOUS flags must be set and MAP_FIXED is optional?

Reply:
Now that I read it again, indeed, it's rather confusing. I reworded it to hopefully be more clear.